local.env

```diff
@@ -3,7 +3,7 @@ export DEBUG=1
 export MEDIA_USE_S3=0
 export SECRET_KEY='dfdfeferewrewrewrrerewr'
 export DB_HOST=localhost
-export DB_PORT=5432
+export DB_PORT=25
```

Comment:
The standard port for postgres is 5432, why do we need to change it? It would not be possible for docker to start a container exposing port 25 on most machines as its a privileged port.

Why do we need to change this?